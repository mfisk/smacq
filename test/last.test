#!/bin/sh

bin/smacqq -m -O <<EOF 2>/dev/null | sort -n
print ts, srcip, dstip from (last srcip, dstip from pcapfile($TEST/0325@1112-snort.pcap))
EOF
