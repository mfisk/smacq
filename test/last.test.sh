#!/bin/sh

$SMACQQ -g -f - <<EOF | sort -n
print ts, srcip, dstip from (last srcip, dstip from pcapfile(0325@1112-snort.pcap))
EOF
