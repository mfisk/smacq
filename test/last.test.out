954011609.363329	192.168.0.54	192.168.0.210
954011731.922436	192.168.0.200	192.168.0.54
954011753.354234	192.168.0.54	192.168.0.200
